.github/workflows/misc_0.yml

```diff
@@ -0,0 +1,125 @@
+# Do not edit this file.
+# This file is generated automatically by executing tox -e generate-workflows
+
+name: Misc 0
+
+on:
+  push:
+    branches-ignore:
+    - 'release/*'
+  pull_request:
+env:
+  CORE_REPO_SHA: main
```

Comment:
I'm wondering if we need to specify a different SHA than the main. Do we need to update the template and generate all workflows again right?

Reply:
Yes, this is a valid concern I had forgottent to address. Not the template. We would need to manually change the SHA in the `.yml` file. I added a condition that will allow us to skip the `generate-workflows` test if there is a label that indicates so just like `Skip changelog` does.